src/go/rdctl/pkg/autostart/autostart_windows.go

```diff
@@ -0,0 +1,83 @@
+package autostart
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+const regPath = `C:\Windows\system32\reg.exe`
```

Comment:
Please check with @dpmerrill, I think he may be pulling in more native registry access stuff for the prefs/profiles work.

Reply:
He is, but that will be for NodeJS, not GoLang.

Ideally the "system directory" should be determined via a [Win32 API call](https://pkg.go.dev/golang.org/x/sys/windows#GetSystemDirectory), but at least using `%SystemRoot%\System32` would be preferable over hardcoding `C:\Windows`.

Reply:
Access to Windows registry seems to be part of the standard Go library: https://pkg.go.dev/golang.org/x/sys/windows/registry

Reply:
I've changed things to use that library. It's cleaner now 👍